elements/bulbs-video/elements/rail-player/components/root.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+
+import Revealed from '../../../components/revealed';
+
+import RailPlayerHeader from './header';
+import RailPlayerCampaign from './campaign';
+
+export default class Root extends React.Component {
+  render () {
+    if (!this.props.video) {
+      return <div/>;
+    }
+
+    return (
+      <div className='rail-player'>
+        <div className='rail-player-header'>
+          <RailPlayerHeader {...this.props}/>
+
+          <div className='rail-player-logo'>
+            Video
+          </div>
+
+          <a
+            className='rail-player-recirc-link'
+            target='_blank'
+            href={this.props.recircUrl}
+          >
+            Watch More
+          </a>
+        </div>
+
+        <div className='rail-player-video bulbs-video-root player'>
+          <Revealed disableSharing={true} {...this.props}/>
```

Comment:
@collin, we'll need to pass in the muted property so this is true: https://github.com/theonion/bulbs-elements/blob/master/elements/bulbs-video/components/revealed.js#L105